src/components/MarkdownProvider/components/CodeExample/index.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, useMemo, useEffect } from 'react';
+import styled, { css, DefaultTheme } from 'styled-components';
+import { ThemeProvider, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { Tooltip } from '@zendeskgarden/react-tooltips';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { ReactComponent as MarkupStroke } from '@zendeskgarden/svg-icons/src/16/markup-stroke.svg';
+import { ReactComponent as CopyStroke } from '@zendeskgarden/svg-icons/src/16/copy-stroke.svg';
+import { ReactComponent as LightningBoltStroke } from '@zendeskgarden/svg-icons/src/16/lightning-bolt-stroke.svg';
+import { ReactComponent as DirectionLtrStroke } from '@zendeskgarden/svg-icons/src/16/direction-ltr-stroke.svg';
+import { ReactComponent as DirectionRtlStroke } from '@zendeskgarden/svg-icons/src/16/direction-rtl-stroke.svg';
+
+import { SyntaxHighlighter } from './components/SyntaxHighlighter';
+import { retrieveCodesandboxParameters } from './utils/retrieveCodesandboxParameters';
+import { copyToClipboard } from './utils/copyToClipboard';
+
+const StyledExampleWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: ${p => p.theme.space.lg};
```

Comment:
Is it okay to not calculate spacing using `theme.space.base` in this project?

Reply:
Yes, it is actually desirable to use the full complement of available Garden theming. Using `space.base` provides leverage at the source component level. The expanded values provide convenience at the consumer level.

Reply:
Thanks for the clarification @jzempel 👍 .